tests/acceptance/features/apiSpaces/copySpaces.feature

```diff
@@ -279,3 +279,175 @@ Feature: copy file
+    Given the administrator has given "Brian" the role "Space Admin" using the settings api
+    And user "Brian" has created a space "Project" with the default quota using the GraphApi
+    And user "Brian" has shared a space "Project" to user "Alice" with role "<role>"
+    And user "Alice" has created folder "folder1"
+    And user "Alice" has uploaded file with content "some content" to "folder1/demo.txt"
+    When user "Alice" copies folder "folder1" from space "Personal" to "folder1" inside space "Project" using the WebDAV API
+    Then the HTTP status code should be "<status-code>"
+    And for user "Alice" the space "Project" <shouldOrNot> contain these entries:
+      | folder1/demo.txt |
+    Examples:
+      | role    | shouldOrNot | status-code |
+      | manager | should      | 201         |
+      | editor  | should      | 201         |
+      | viewer  | should not  | 403         |
+
+
+  Scenario Outline: User copies a folder from space personal to space shares jail with permission editor
+    Given user "Brian" has created folder "/testshare"
+    And user "Brian" has shared folder "/testshare" with user "Alice" with permissions "<share-permission>"
+    And user "Alice" has accepted share "/testshare" offered by user "Brian"
+    And user "Alice" has created folder "folder1"
+    And user "Alice" has uploaded file with content "some content" to "folder1/demo.txt"
+    When user "Alice" copies folder "folder1" from space "Personal" to "/testshare/folder1" inside space "Shares Jail" using the WebDAV API
+    Then the HTTP status code should be "<status-code>"
+    And for user "Alice" folder "testshare" of the space "Shares Jail" <shouldOrNot> contain these files:
+      | folder1/demo.txt |
+    Examples:
+      | share-permission | shouldOrNot | status-code |
+      | 31               | should      | 201         |
+      | 17               | should not  | 403         |
+
+
+  Scenario Outline: User copies a folder from space shares jail with different role to space personal
+    Given the administrator has given "Brian" the role "Space Admin" using the settings api
+    And user "Brian" has created folder "/testshare"
+    And user "Brian" has uploaded file with content "testshare content" to "/testshare/testshare.txt"
+    And user "Brian" has shared folder "/testshare" with user "Alice" with permissions "<permissions>"
+    And user "Alice" has accepted share "/testshare" offered by user "Brian"
+    When user "Alice" copies file "/testshare/testshare.txt" from space "Shares Jail" to "testshare.txt" inside space "Personal" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And for user "Alice" the space "Personal" should contain these entries:
+      | /testshare.txt |
+    And for user "Alice" the content of the file "/testshare.txt" of the space "Personal" should be "testshare content"
+    Examples:
+      | permissions |
```

Comment:
make this consistent in every scenarios. somewhere its share-permission and somewhere its permission